mvp-spec/partials/transmission-request.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": "0.1",
+    "seed": {
+        "version": "0.1",
+        "transaction_ids": [ 
+            "4640dc9f-385f-4e02-a0e5-abbf241af94d", 
+            "7d71a23a-fafa-449a-8b85-63a634780107" 
+        ],
+        "publisher": "publisher.com",
+        "source": {
+            "domain": "publisher.com",
+            "timestamp": 1639582000,
+            "signature": "f1f4871d48b825931c5016a433cb3b6388f989fac363af09b9ee3cd400d86b74"
+        }
+    },
+    "data": {
+        "identifiers": [
+            {
+                "version": "0.1",
+                "type": "paf_browser_id",
+                "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+                "source": {
+                    "domain": "operator0.com",
+                    "timestamp": 1639580000,
+                    "signature": "868e7a6c27b7b7fe5fed219503894bf263f31bb6d8fd48336d283e77b512cda7"
+                }
+            }
+        ],
+        "preferences": {
+            "version": "0.1",
+            "data": { 
+                "use_browsing_for_personalization": true 
+            },
+            "source": {
+                "domain": "cmp1.com",
+                "timestamp": 1639581000,
+                "signature": "65acdcfdbdba8b17936f25a32b33b000393c866588d146cb62ec51ab8890c54f"
+            }
+        }
+    },
+    "source": {
+        "domain": "dsp1.com",
```

Comment:
Technically this is OK, but it's a bit confusing that this transmission request example shows signature from dsp1.com.
Typically DSPs, won't send transmission requests.

In addition, parents is empty so it's expected that the signature is from the publisher.com (like the seed)